SourceS/file_util.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#if _POSIX_C_SOURCE >= 200809L
+#include <unistd.h>
+#else
+#include <cstdio>
+#endif
+
+namespace dvl {
+
+inline bool FileExists(const char *path)
+{
+#if _POSIX_C_SOURCE >= 200809L
+	return access(path, F_OK) == 0;
+#else
+	auto *file = std::fopen(name.c_str(), "rb");
+	if (file) {
+		fclose(file);
+		return true;
+	} else {

```

Comment:
I would prefer and early return instead:
```
	if (!file) {
		return false;
	}
	fclose(file);
	return true;
```

Reply:
Done